docs/platforms/python/tracing/instrumentation/custom-instrumentation/index.mdx

```diff
@@ -145,21 +173,21 @@ sentry_sdk.init(
 
 Now, whenever a function specified in `functions_to_trace` will be executed, a span will be created and attached as a child to the currently running span.
 
-<Alert level="warning" title="Important">
+<Alert title="Important" level="info">
 
 To enable performance monitoring for the functions specified in `functions_to_trace`, the SDK needs to load the function modules. Be aware, there may be code being executed in modules during module loading. To avoid this, use the method described above to trace your functions.
 
 </Alert>
 
 ## Accessing the Current Transaction
```

Comment:
I think we could probably also delete this section or move it to its own page. This is probably too much information for most use cases, especially because users who are doing custom instrumentation should already have a reference to the current transaction, since `start_transaction` returns the transaction.